Cargo.toml

```diff
@@ -106,6 +111,7 @@ time = { version = "0.3.37", features = [
     "formatting",
     "parsing",
 ] }
+tokio = { version = "1.42.0", features = ["full"] }
```

Comment:
We made an architectural decision to use synchronous rather than async Rust and I think that has been working well overall. I would like to continue with that, so I would request that we implement this feature using synchronous Rust.

Reply:
This isn't really possible as the actix-web library uses async (as do the majority of Rust http server libraries).

Reply:
Going to resolve this so that we can have the rest of this convo below when talking specifically about rouille, as it's the only reasonably maintained non-async web server and has its own issues worth discussing.